src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
 
+        Machine[] machines = new Machine[]{excavator,bulldozer,truck};
```

Comment:
There's an unnecessary comma after the last element of the array. While this is syntactically correct in Java, it's cleaner to remove the trailing comma for consistency.